### PR TITLE
feat: [PL-25250]: Added basic integration for chaos module

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
##### Summary:

- Added routing from parent to child microfrontend.
- Added project selection and routing associated with proper project selection
- Added Access control for ChaosHub resource
- Added test cases for the new 75-chaos module

##### Jira Links:

https://harness.atlassian.net/browse/PL-25250

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by tan-nhu — APPROVED** [review]

**Comment by SahilKr24** [comment]
retrigger titlecheck

**Comment by SahilKr24** [comment]
trigger rebase

File: src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      >
+        /account/dummy/chaos/orgs/org/projects/project/dashboard
+      </div>
+    `)
+  })
+
+  test('should go to access control when selected from the sidebar', async () => {
+    render(
+      <TestWrapper
+        path="/account/:accountId/chaos/orgs/:orgIdentifier/projects/:projectIdentifier"
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx:61]: you can use `path={toChaosMicroFrontend({accountId: ':accountId` ... })}`

File: src/modules/75-chaos/pages/home/images/chaos.svg [hunk 2]
```diff
@@ -0,0 +1,970 @@
+<svg width="1445" height="590" viewBox="0 0 1445 590" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/pages/home/images/chaos.svg:1]: lets run it through https://jakearchibald.github.io/svgomg/

File: src/modules/75-chaos/strings/strings.en.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+homepage:
+  chaosHomePageTitle: Chaos Engineering
+  slogan: Take your software build process to the next level using our Harness Chaos Engineering.
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/strings/strings.en.yaml:3]: let's get it checked by the copy team

File: src/modules/75-chaos/pages/home/images/chaos.svg (hunk 2 above)
Comment (arkajyotiMukherjee) [reply src/modules/75-chaos/pages/home/images/chaos.svg]: done

**Comment by arkajyotiMukherjee** [comment]
retrigger jest

**Comment by SahilKr24** [comment]
retrigger coverage

**Review by deepesh-ui — APPROVED** [review]

File: src/modules/75-chaos/strings/strings.en.yaml (hunk 3 above)
Comment (arkajyotiMukherjee) [reply src/modules/75-chaos/strings/strings.en.yaml:3]: Got the updated slogan...we'll update the string in the following PR

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request closed without merging**